### DISCUSSION TITLE
Frigate errors which consume max CPU -  Uneble to create jpg / errors with detecting

### DISCUSSION THREAD

**Discussion by Engine1992** [blakeblackshear/frigate#5592] [Ask A Question]
Hi! I runnking Frigate with 2 G Corals. I have 21 cameras on it. Everything is fine but i have some errors in loop:

`[2023-02-26 09:33:49] frigate.object_processing      WARNING : Unable to create jpg because frame 1677371545.376247 is not in the cache`
`[2023-02-26 09:51:56] ffmpeg.Firma_Hala_lewa_1.detect ERROR   : rtsp://usr:pass@IP:PORT/h264Preview_01_sub: corrupt decoded frame in stream 0`
`[2023-02-26 09:51:56] ffmpeg.Firma_Hala_lewa_1.detect ERROR   : [h264 @ 0x55d2895bfc00] cabac decode of qscale diff failed at 23 4`
`[2023-02-26 09:51:56] ffmpeg.Firma_Hala_lewa_1.detect ERROR   : [h264 @ 0x55d2895bfc00] error while decoding MB 23 4, bytestream -5`

Where can be problem? The Frigate consume max power of CPU. Detection on Corals works OK.

**Comment by blakeblackshear** [comment]
Pleas open a support issue rather than starting a discussion.
